index.html

```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
+                    </p>
+                </nav>
+                <nav class="grid-image">
+                        <nav class="second-image">  
+                            <img src="icon/illustration 2-about me mobile.svg">
+                        </nav>  
+                        <nav class="resume_button"> 
+                            <nav class="grid-item resume">
+                                <button class="button-resume" type="button">Get my resume</button>
+                            </nav>
+                        </nav>
+                </nav>
+            </nav>
+            <nav class="grid-container-ab">
+                <nav class="grid-language">
+                    <img src="icon/icon-languages.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Languages</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-about1"><h4>JavaScript</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>HTML</h4></nav>
+                    <nav class="flex-about flex-about4"><h4>CSS</h4></nav>
+                </nav>
+            </nav>
+            <nav class="grid-container-ab">
+                <nav class="grid-language">
+                    <img src="icon/icon-frameworks.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Frameworks</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutjs"><h4>React.js</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby on Rails</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>RSpec</nav>
+                </nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutCapy"><h4>Capybara</h4> </nav>
+                    <nav class="flex-about flex-about6"><h4>Selenium</h4></nav>
+                </nav>
+            </nav>
+            <nav class="grid-container-ab">
+                <nav class="grid-language">
+                    <img src="icon/icon-skills.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Skills</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutdb"><h4>Database Managment</h4> </nav>
+                    <nav class="flex-about flex-aboutvs"><h4>Visual Control</h4></nav>
+                </nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutcli"><h4>CLI</h4> </nav>
+                    <nav class="flex-about flex-aboutweb"><h4>Webdevelopment</h4></nav>
+                    <nav class="flex-about flex-aboutapi"><h4>API Design</h4></nav>
+                </nav>
+            </nav>
```

Comment:
- I can see you are using `nav` tag everywhere. It is a serious semantic error. `nav` tag is specifically for `navigation menu` of an webpage.
- There are other dedicated tags for specific sections such as `div`, `ul`, `li`.
- [ ] Kindly use dedicated `tags` for each section.

***[Hints]*** I have mentioned some corrections for `ul` and `li` tags. Other than `ul` and `li` tags replace all `nav` tags with `div` tag.

Reply:
Great

Reply:
Thanks a lot